### DISCUSSION TITLE
Protractor to Webdriver IO Conversion

### DISCUSSION THREAD

**Discussion by anuvratashu** [webdriverio/codemod#52] [Migration Issue]
Hi
i was trying to the migration but the migration is failing with the following error:
![image](https://user-images.githubusercontent.com/14295329/141942816-52fe0eb8-4662-4ebc-ac28-2ba4af9a68b7.png)

**Comment by christian-bromann** [comment]
Can you check the file that is mentioned in the stack trace? `EdgeManagementSpec.ts` line 26?
